<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>671</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_pos">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="pushButton_neg">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_sign">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x^</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_sub">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A-B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_time">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A*B</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_add">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>A+B</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton_point">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="pushButton_deriv">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>求导</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_value">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>求值</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>单击以输入B</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="pushButton_back">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退格</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_print">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>标准输出</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="pushButton_exit">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>多项式A(第一个操作数/求导/求值/标准输出)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>多项式B(第二个操作数/x的值)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
